Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -0,0 +1,640 @@
+            # API returns a number, we add the time format to it
+            context_output['HalfLife'] = str(context_output['HalfLife']) + ' Days'
+
+        human_readable += tableToMarkdown('Observable ID {} related entities'.format(observable_id), context_output)
+
+        test_mechanisms_output = []
+
+        for mechanism in test_mechanisms:
+
+            mechanism_output = {
+                'Type': mechanism.get('test_mechanism_type')
+            }
+
+            mechanism_rules = mechanism.get('rules')
+
+            mechanism_rules_outputs = []
+
+            for rule in mechanism_rules:
+                mechanism_rules_outputs.append(rule.get('value'))
+
+            mechanism_output['Rule'] = mechanism_rules_outputs
+
+            test_mechanisms_output.append(mechanism_output)
+
+        if test_mechanisms_output:
+            context_output['TestMechanism'] = test_mechanisms_output
+            human_readable += tableToMarkdown('Test mechanisms', test_mechanisms_output, removeNull=True)
+
+        sources = entity.get('sources')
+
+        sources_output = []
+
+        for source in sources:
+            sources_output.append({
+                'Name': source.get('name'),
+                'Type': source.get('source_type'),
+                'Reliability': source.get('source_reliability')
+            })
+
+        if sources_output:
+            context_output['Source'] = sources_output
+            human_readable += tableToMarkdown('Sources', sources_output, removeNull=True)
+
+        exposure = entity.get('exposure')
+
+        exposure_output = {
+            'Exposed': True if exposure.get('exposed') is True else False,
+            'Detection': True if exposure.get('detect_feed') is True else False,
+            'Prevention': True if exposure.get('prevent_feed') is True else False,
+            'Community': True if exposure.get('community_feed') is True else False,
+            'Sighting': True if exposure.get('sighted') is True else False
+        }
+
+        context_output['Exposure'] = exposure_output
+        human_readable += tableToMarkdown('Exposure', exposure_output, removeNull=True)
+
+        context_outputs.append(context_output)
+
+    context = {
+        'EclecticIQ.Entity(val.ID===obj.ID)': context_outputs
+    }
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': response,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': context
+    })
+
+
+def processed_extract(observable_id):
+    cmd_url = '/private/entities/processed-extract/{}'.format(observable_id)
+    response = http_request('GET', cmd_url)
+    return response
+
+
+def original_extract(observable_id):
+    cmd_url = '/private/entities/original-extract/{}'.format(observable_id)
+    response = http_request('GET', cmd_url)
+    return response
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+login()
```

Comment:
shouldn't be a logout?

Reply:
i guess there should be, but their api does not support it